Tetris Clone C++/TetrisPiece.hpp

```diff
@@ -33,6 +33,7 @@ class TetrisPiece
     int x, y;
     // col, row are grid-based positions
     int col, row;
+    int * frameCounter;
```

Comment:
the frameCounter could exist inside the tetris pieces since it seems to be the only use for it.